app/voip/server/lib/Helper.ts

```diff
@@ -0,0 +1,165 @@
+import { Meteor } from 'meteor/meteor';
+import { Match, check } from 'meteor/check';
+
+
+import { IVisitor } from '../../../../definition/IVisitor';
+import { VoipRoom } from '../../../models/server/raw';
+import { settings } from '../../../settings/server';
+import { Subscriptions, Users } from '../../../models/server';
+import { Logger } from '../../../../server/lib/logger/Logger';
+
+const logger = new Logger('VoipHelper');
+
+
+const emailValidationRegex = /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
```

Comment:
I'm pretty sure we have a setting for email validation already

Reply:
Does not exist anymore